docs/subscriptions.md

```diff
@@ -1,28 +1,86 @@
+  namespace: openshift-operators
 spec:
   channel: stable
   name: my-operator
-  source: redhat-operators 
+  source: redhat-operators
   sourceNamespace: openshift-marketplace
+  approval: Manual
+```
+
+Setting the `approval` field to manual will prevent OLM from automatically installing the operator. As such, you will need to approve the installPlan which can be done with the following commands:
+
+```bash
+kubectl -n openshift-operators get installplans
+NAME            CSV                                APPROVAL   APPROVED
+install-bfmxd   my-operator.v0.1.0                 Manual     false
+
+$ kubectl -n default patch installplan install-bfmxd -p '{"spec":{"approved":true}}' --type merge
+installplan.operators.coreos.com/install-bfmxd patched
+
+$ kubectl -n openshift-operators get installplans
+NAME            CSV                                APPROVAL   APPROVED
+install-bfmxd   my-operator.v0.1.0                 Manual     true
+```
+
+Now that the `install-bfmxd` installPlan is in the approved state, OLM will install the operator defined by the `my-operator.v0.1.0` CSV.
+
+When the CatalogSource is updated with a newer version of that operator in the channel you selected, a new installPlan will be created in the namespace that you installed the operator to, as shown below:
+
+```bash
+$ kubectl -n openshift-operators get installplans
+NAME            CSV                                APPROVAL   APPROVED
+install-bfmxd   my-operator.v0.1.0                 Manual     true
+install-svojy   my-operator.v0.2.0                 Manual     false
+```
+
+From here, you can approve `install-svojy` using the patch command shown earlier:
+
+```bash
+kubectl -n openshift-operators get installplans
+NAME            CSV                                APPROVAL   APPROVED
+install-bfmxd   my-operator.v0.1.0                 Manual     true
+install-svojy   my-operator.v0.2.0                 Manual     false
+
+$ kubectl -n openshift-operators patch installplan install-svojy -p '{"spec":{"approved":true}}' --type merge
+installplan.operators.coreos.com/install-svojy patched
+
+$ kubectl -n openshift-operators get installplans
+NAME            CSV                                APPROVAL   APPROVED
+install-bfmxd   my-operator.v0.1.0                 Manual     true
+install-svojy   my-operator.v0.2.0                 Manual     true
 ```
 
-This Subscription object defines the name and namespace of the operator, as well as the catalog from which the operator
-data can be found. The channel (such as alpha, beta, or stable) helps determine which version of the operator should be installed
-from the catalog source. 
+With the new installPlan in the approve state, the `my-operator.v0.2.0` CSV will be deployed to the cluster and the resources installed as a result of the `my-operator.v0.1.0` CSV will be updated or deleted. Notice that OLM will not delete old installPlans as they act as a record of CSVs that were installed on your cluster.
```

Comment:
Maybe another note that ensures people such as:
```suggestion
With the new installPlan in the approve state, the `my-operator.v0.2.0` CSV will be deployed to the cluster and the resources installed as a result of the `my-operator.v0.1.0` CSV will be updated or deleted. Notice that OLM will not delete old installPlans as they act as a record of CSVs that were installed on your cluster. On the other hand, if update fails, the old version of the operator will not be deleted.
```

Reply:
Is this true @ecordell? Does OLM perform rollback if a new CSV install fails?